napalm_ios/ios.py

```diff
@@ -354,6 +354,8 @@ def commit_config(self):
                 raise ReplaceConfigException("Candidate config file does not exist")
             if self.auto_rollback_on_error:
                 cmd = 'configure replace {} force revert trigger error'.format(cfg_file)
+            elif confirmed:
+                cmd = 'configure replace bootflash:{} force time {}'.format(cfg_file, confirmed)
```

Comment:
Don't hard-code the file system `bootflash`. The `cfg_file` variable already has the file_system which either gets auto detected or set by optional_args.